src/components/sections/Footer.svelte

```diff
@@ -70,6 +70,8 @@
                 &bull;
                 <a href="code-of-conduct">Code of Conduct</a>
                 &bull;
+                <a href="privacy-policy">Privacy Policy</a>
```

Comment:
```suggestion
                <a href="/privacy-policy">Privacy Policy</a>
```

Powyżej i poniżej też.
Dodatkowo zauważyłem, że i w Menu komponencie też warto dodać tego slesza ;)